utils/cache.js

```diff
@@ -12,7 +12,27 @@ exports.currentStatus = {
 };
 
 // Series of get calls to retrieve current state of cache
-exports.nytCounties = () => this.currentStatus.nytCounties;
+/**
+ * Parses NYT Counties data
+ * @param 	{string}	lastdays  	How many days to show always take lastest
+ * @returns {Array}				The filtered historical data.
+ */
+exports.nytCounties = (lastdays = 30) => {
+	if (lastdays === 'all') {
+		return this.currentStatus.nytCounties;
+	} else {
+		var priorDate = new Date();
```

Comment:
Can you do
```js
const priorDate = new Date().setDate(priorDate.getDate() - lastdays).toISOString().slice(0, 10);
```
It's cleaner and always prefer `const` over `var`.

Reply:
I tried this but seems like priorDate is not declared yet so this `priorDate.getDate()` block won't work. So had to keep it the previous way and let it be var as we cannot assign values to a const.